FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.Core.UnitTests/DataProcessor/ParquetDataProcessorTests.cs

```diff
@@ -123,7 +123,8 @@ public async Task GivenAValidInputData_WhenProcessWithCustomizedSchema_CorrectRe
         public async Task GivenAValidMultipleLargeInputData_WhenProcess_CorrectResultShouldBeReturned()
         {
             var largePatientSingleSet = TestUtils.LoadNdjsonData(Path.Combine(TestUtils.TestDataFolder, "Large_Patient.ndjson"));
-            var largeTestData = Enumerable.Repeat(largePatientSingleSet, 100).SelectMany(x => x);
+            // Maximum row number for arrow cpp parser is 100000, see https://github.com/apache/arrow/blob/42a9b32141c3c5a7178bef6644872d14f3051ce6/cpp/src/arrow/json/parser.h#L51
+            var largeTestData = Enumerable.Repeat(largePatientSingleSet, 10).SelectMany(x => x);
```

Comment:
Why the parquet conversion was successfully before? Is this a linux limitation?

Reply:
Because the block size used in earlier tests are default value, which is small and should not reach the limit